### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by petrparolek**
- polished docs
- travis -> GitHub Actions
- updated ninjify/qa

### CODE REVIEW

**Comment by f3l1x** [comment]
👍 by me. WDYT @jiripudil?

File: README.md [hunk 1]
```diff
@@ -1,43 +1,52 @@
...
-        <br/>
-        <a href="https://github.com/jiripudil">Jiří Pudil</a>
-      </td>
-    </tr>
-  </tbody>
-</table>
+
+## Development
+
+See [how to contribute](https://contributte.org) to this package. This package is currently maintained by these authors.
```
Comment (jiripudil) [inline README.md:43]:
```suggestion
See [how to contribute](https://contributte.org/contributing.html) to this package. This package is currently maintained by these authors.
```

File: README.md [hunk 2]
```diff
@@ -1,43 +1,52 @@
...
+
+<a href="https://github.com/jiripudil">
+    <img width="80" height="80" src="https://avatars2.githubusercontent.com/u/1042159?v=3&s=80">
+</a>
+
+-----
+
+Consider to [support](https://contributte.com/partners) **contributte** development team.
```
Comment (jiripudil) [inline README.md:51]:
```suggestion
Consider to [support](https://contributte.org/partners.html) **contributte** development team.
```

**Review by jiripudil — CHANGES_REQUESTED** [review]
I believe some of the links in readme are wrong. Apart from that, lgtm :+1:

File: README.md (hunk 1 above)
Comment (petrparolek) [reply README.md:43]: @f3l1x whick link is OK?

File: README.md (hunk 2 above)
Comment (petrparolek) [reply README.md:51]: @f3l1x whick link is OK?

**Comment by petrparolek** [comment]
I do not know which links are OK. I made readme by copy and paste from another contritutte packages.

**Comment by jiripudil** [comment]
Oh, then I guess they're incorrect in other contributte packages 🤔 I've taken the suggested links from [contributte/bare](https://github.com/contributte/bare)

**Comment by f3l1x** [comment]
Great catch @jiripudil. You're correct. Could you please @petrparolek update the links? And update these links in other packages as well, it should be easy change. Thank you.

**Comment by petrparolek** [comment]
tjhhanks, update in this PR

**Comment by jiripudil** [comment]
@petrparolek please check the links for typos like [this](https://github.com/contributte/validator/pull/3/commits/3713018da469fbc05fb95cddc582cc04a4219d99#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L51) before you update them in other repositories

**Review by jiripudil — APPROVED** [review]

**Pull request merged**